pkg/collect/host_kernel_modules.go

```diff
@@ -114,20 +121,16 @@ func (c *CollectHostKernelModules) Collect(progressChan chan<- interface{}) (map
 type kernelModulesLoadable struct{}
 
 // collect the list of modules that can be loaded by the kernel.
-func (l kernelModulesLoadable) collect() (map[string]KernelModuleInfo, error) {
+func (l kernelModulesLoadable) collect(kernelRelease string) (map[string]KernelModuleInfo, error) {
 	modules := make(map[string]KernelModuleInfo)
 
-	out, err := exec.Command("uname", "-r").Output()
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to determine kernel release")
-	}
-	kernel := strings.TrimSpace(string(out))
-
-	kernelPath := "/lib/modules/" + kernel
-
+	kernelPath := filepath.Join("/lib/modules", kernelRelease)
 	if _, err := os.Stat(kernelPath); os.IsNotExist(err) {
-		klog.V(2).Infof("modules are not loadable because kernel path %q does not exist, assuming we are in a container", kernelPath)
```

Comment:
It might be worth logging in this instance that `/lib/modules` does not exist, checking `/usr/lib/modules` before the `assuming we are in container` log fires, because that makes it sound like we only ever checked `/usr/lib/modules`.

Or maybe update the `kernelPath` in `Infof` to be:

```go
klog.V(2).Infof("modules are not loadable because kernel path %q or %q do not exist, assuming we are in a container", kernelPathLib, kernelPathUsr)
```